hexrd/ui/interactive_template.py

```diff
@@ -27,11 +23,48 @@ def __init__(self, parent=None):
         self.translation = [0, 0]
         self.complete = False
         self.event_key = None
-        self.parent.setFocusPolicy(Qt.ClickFocus)
+        self.detector = detector
+        self.instrument = instrument
+        self._static = True
+
+        for canvas in self.image_tab_widget.active_canvases:
+            canvas.setFocusPolicy(Qt.ClickFocus)
+
+    @property
+    def current_canvas(self):
+        idx = self.image_tab_widget.current_index
+        return self.image_tab_widget.active_canvases[idx]
+
+    @property
+    def ax(self):
+        idx = HexrdConfig().current_imageseries_idx
+        return self.current_canvas.axes_images[idx]
```

Comment:
I am under the impression that this might not be correct. Isn't the `current_imageseries_idx` used for viewing different frames in an unaggregated image series? But those `axes_images` are instead the separate images for each detector in the untabbed raw view.

Reply:
You are correct, thanks for catching this! This was in fact the reason for the rotation bug as well. Thanks!